spec/controllers/application_controller_spec.rb

```diff
@@ -21,7 +21,7 @@
       describe '#search_facets_url' do
```

Comment:
Should we update this string?

Reply:
👍 ; it's nice that the string was wrong in the first place.

Reply:
Yup!